hawkular-alerts-rest-tests/pom.xml

```diff
@@ -353,7 +437,14 @@
                 </goals>
                 <configuration>
                   <includes>
-                    <include>**/*ITest*</include>
+                    <include>**/ActionsITest*</include>
+                    <include>**/AlertsITest*</include>
+                    <include>**/ConditionsITest*</include>
+                    <include>**/DampeningITest*</include>
+                    <include>**/EventsITest*</include>
+                    <include>**/EventsLifecycleITest*</include>
+                    <include>**/LifecycleITest*</include>
+                    <include>**/TriggersITest*</include>
```

Comment:
For some reason, the exclusion rule (which is more simple) it doesn't work well on this plugin.
I'm going to re-check but that was the main reason.
Putting a excludes rules didn't work.